manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -216,6 +216,10 @@ public function registerContainerConfiguration(LoaderInterface $loader): void
         if ($servicesFile = $this->getConfigFile('services.yml')) {
             $loader->load($servicesFile);
         }
+
+        if (is_dir($this->getProjectDir().'/src')) {
```

Comment:
This condition is also true when unit testing the manager-bundle separately, therefore the bundles test fails: https://github.com/contao/contao/pull/1165/checks?check_run_id=393653023#step:4:888

<img width="923" alt="" src="https://user-images.githubusercontent.com/1192057/72563819-c2affd80-38ae-11ea-8182-7ae6da0604df.png">

Reply:
When testing the bundle selaratey, its dependencies should still be installed? So core-bundle should be there as well as SecurityBundle ??